io.github.igorlogius.scr2ppm.yml

```
@@ -0,0 +1,25 @@
+app-id: io.github.igorlogius.scr2ppm
+runtime: org.freedesktop.Platform
+runtime-version: '21.08'
+sdk: org.freedesktop.Sdk
+command: scr2ppm
+rename-icon: scr2ppm
+rename-desktop-file: scr2ppm.desktop
+rename-appdata-file: scr2ppm.metainfo.xml
+finish-args:
+  - --socket=x11
+  - --share=ipc
+modules:
+  - name: scr2ppm
+    buildsystem: simple
+    build-commands:
+      - cc scr2ppm.c -lX11 -lxcb -lXau -lXdmcp -o scr2ppm
+      - install -D scr2ppm /app/bin/scr2ppm
+      - install -D scr2ppm.desktop /app/share/applications/scr2ppm.desktop
+      - install -D scr2ppm.metainfo.xml /app/share/metainfo/scr2ppm.metainfo.xml
+      - install -D scr2ppm.png /app/share/icons/hicolor/128x128/apps/scr2ppm.png
+    sources:
+      - type: git
+        url: https://github.com/igorlogius/scr2ppm.git
+        branch: main
+        commit: a511f2f070161410d27e8b83c0c95bd911e0c46f
```

```suggestion
        commit: a511f2f070161410d27e8b83c0c95bd911e0c46f
```

Because branches move, if the branch is updated the build is broken. Pining a commit is the right way to go.